web/modules/custom/collection/src/ContextProvider/CollectionRouteContext.php

```diff
@@ -88,20 +87,30 @@ public function getRuntimeContexts(array $unqualified_context_ids) {
     $cacheability = new CacheableMetadata();
     $cacheability->setCacheContexts(['route']);
 
-    $context = new Context($context_definition, $value);
+    $collection_context_definition = new ContextDefinition('entity:rdf_entity:collection', NULL, FALSE);
+    $context = new Context($collection_context_definition, $value);
     $context->addCacheableDependency($cacheability);
-
     $result['collection'] = $context;
 
+    // Also create a context definition to be available for the og_menu block.
+    $collection_context_definition = new ContextDefinition('entity', $this->t('Organic group provided by collection'), FALSE);
+    $context = new Context($collection_context_definition, $value);
+    $context->addCacheableDependency($cacheability);
+    $result['og'] = $context;
```

Comment:
This is strange, we have two identical contexts now? That is not necessary. We should either provide the 'collection' context, or the 'og' context, but not both.

I think now that we have the 'og' context which is provided by OG Menu, that we probably can get rid of our 'collection' context, and use 'og'.